pkg/cri/server/container_update_resources.go

```diff
@@ -19,7 +19,6 @@
 package server
 
 import (
-	"context"
 	gocontext "context"
```

Comment:
`context` should be preferred over `gocontext`

Reply:
Thanks for @AkihiroSuda
Maybe we need to discuss it again and come to an agreement @Iceber 

<img width="683" alt="image" src="https://github.com/containerd/containerd/assets/47143494/8001972e-8f42-4fb8-bcd6-7e3437062e1b">

Reply:
> There is no name conflict here, so it might be better to use "context"

My meaning is to use `context`, not `gocontext`.